packages/deployments/contracts/contracts/core/connext/facets/RoutersFacet.sol

```diff
@@ -564,7 +564,12 @@ contract RoutersFacet is BaseConnextFacet {
     if (_amount == 0) revert RoutersFacet__addLiquidityForRouter_amountIsZero();
 
     // Get the canonical asset ID from the representation.
-    (TokenId memory canonical, bytes32 key) = _getApprovedCanonicalId(_local);
+    // NOTE: not using `_getApprovedCanonicalId` because candidate can *only* be local
+    TokenId memory canonical = s.representationToCanonical[_local];
```

Comment:
`_local` can be a canonical asset on its canonical domain which then `s.representationToCanonical[_local]` will not be set (is the default value). In this case the values for `canonical` and `key` will not be correctly calculated.